docs/sphinx/install.rst

```diff
@@ -143,9 +143,8 @@ we refer to the `CUDA Quantum GitHub repository`_.
 
 Next Steps
 ----------
-With the CUDA Quantum Docker image installed and a container up and running, check out the
-Using CUDA Quantum page_. To run the examples codes in the container, checkout the Compiling
-and Executing section here_. 
+With the CUDA Quantum Docker image installed and a container up and running, check out :doc:`Using CUDA Quantum <using/cudaq>`. 
```

Comment:
nitpick:
Naively, I'm not sure what "check out" means exactly here, since it can be an action or slang in English. Does it mean "read some webpage" or actually download some code from a repository? In the former case, the qualifying clause probably isn't needed. In the latter, it's not clear if I should download the code into the container or somewhere else precisely.

Reply:
This section looks a bit outdated in any case. I updated it a bit and rephrased that.